example/package.json

```diff
@@ -8,12 +8,14 @@
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint .",
-    "start:windows": "react-native start --use-react-native-windows --projectRoot ../"
+    "clean:windows": "rmdir /s node_modules\\react-native-orientation-locker\\node_modules\\",
+    "start:windows": "react-native start --use-react-native-windows",
+    "windows": "react-native run-windows --no-packager"
   },
   "dependencies": {
     "react": "16.13.1",
     "react-native": "0.63.2",
-    "react-native-orientation-locker": "link:../",
+    "react-native-orientation-locker": "file: ../../../",
```

Comment:
is this right? it seems to point outside of the repo root

Reply:
Yea it seems weird but it loads the module correctly as a dependency. My best guess is that it wants the relative path from the node_modules folder and then in the directory it lands in it looks for a directory with the name react-native-orientation-locker and adds that directory to node_modules.